src/subscriptions/SubscriptionsContainer.js

```diff
@@ -5,12 +5,25 @@ import type { GlobalState } from '../types';
 import { getAuth } from '../selectors';
 import boundActions from '../boundActions';
 import SubscriptionsCard from './SubscriptionsCard';
+import { getUnreadStreams } from '../baseSelectors';
+import { queueMarkAsRead } from '../api';
 
 export default connect(
   (state: GlobalState) => ({
     auth: getAuth(state),
     streams: state.streams,
     subscriptions: state.subscriptions,
+    markStreamMessagesRead: (streamId: number) => {
```

Comment:
This is a place where we specify what parts of the Redux state we want.
Creating a new function wouldn't go here.